src/components/Tokens/TokenDetails/PriceChart.tsx

```diff
@@ -104,9 +104,7 @@ export function PriceChart({ width, height, prices, timePeriod }: PriceChartProp
 
   // set display price to ending price when prices have changed.
   useEffect(() => {
-    if (prices) {
-      setDisplayPrice(endingPrice)
-    }
```

Comment:
I believe this check is now redundant, and it was causing prices to not update when navigating to a token not on the backend

Reply:
New features in _this_ PR were broken by this. It doesn't fix existing issues